angular14-demo-app/copyright.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for
+ * additional information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+const glob = require('glob');
+const fs = require('fs-extra');
+
+const copyrightHeaderTS = `/*
+ * Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+ *
```

Comment:
2023 ay be retrieved by a function const dateNow = new Date(), year dateNow.year so we would always have the current year